### ISSUE TITLE
Piano Grand Staff

### ISSUE THREAD

**Issue by drschulz** [paulrosen/abcjs#57]
Hi,

First of all, I absolutely love this library. Thank you so much for the time you have put into it. I am currently trying to display a piano grand staff with staff groups. Here is the abc:

```
M: 4/4
L: 1/16
K: A
%%staves {V1 V2}
V: V1 clef=treble
V: V2 clef=bass
[V: V1]C16| D16| D16| D16|
[V: V2]B,,16| B,,16| D,16| D,16|
```

To the best of my knowledge, the curly braces in the staves directive should put a curly brace around the entire stave at the front, as in this example:

http://abcnotation.com/tunePage?a=richardrobinson.tunebook.org.uk/static/tunebook/00100

However, no element is present when I render my abc code:

![image](https://cloud.githubusercontent.com/assets/5833009/13367743/b8eecf8a-dc99-11e5-8a13-7f6be7e4bcea.png)

Is it possible to display the brace for the staves? What would be necessary for me to implement in order to get the brace to display? Thanks!

**Comment by paulrosen** [comment]
Yes, that feature of ABC isn't implemented yet. It is parsed, but the writer ignores it.

Are you asking how you can help to write that code? If so, then I think the place to start looking is ABCJS.write.AbstractEngraver.prototype.createABCStaff. abcstaff might have an element { brace: "start" } or { brace: "end" }. That's how you would know that a brace is requested. I'd be happy to work with you if you email me directly.

**Comment by paulrosen** [comment]
Fixed in version 3.0
